.github/workflows/ci.yaml

```diff
@@ -96,88 +96,94 @@ jobs:
         run: go test -race ./... -timeout 2m
 
   cli:
-    runs-on: ubuntu-latest
-    timeout-minutes: 3
+    strategy:
+      matrix:
+        os: [ubuntu-latest, windows-latest]
 
-    services:
-      postgres:
-        image: postgres
-        env:
-          POSTGRES_PASSWORD: postgres
-        options: >-
-          --health-cmd pg_isready
-          --health-interval 2s
-          --health-timeout 5s
-          --health-retries 5
-        ports:
-          - 5432:5432
+    runs-on: ${{ matrix.os }}
+    timeout-minutes: 10
 
     steps:
+      - uses: ikalnytskyi/action-setup-postgres@v6
+        with:
+          database: river_dev
+          password: postgres
+
       - uses: actions/setup-go@v5
         with:
           go-version: "stable"
           check-latest: true
 
+      - name: Setup GOBIN (Linux)
+        if: runner.os == 'Linux'
+        run: |
+          echo "GOBIN=$(go env GOPATH)/bin" >> $GITHUB_ENV
+          echo "PATH=$(go env GOPATH)/bin:$PATH" >> $GITHUB_ENV
+        shell: bash
+
+      - name: Setup GOBIN (Windows)
+        if: runner.os == 'Windows'
+        run: |
+          $gobin = "$(go env GOPATH)\bin"
+          echo "GOBIN=$gobin" >> $env:GITHUB_ENV
+          echo $gobin | Out-File -FilePath $env:GITHUB_PATH -Encoding utf8 -Append
+        shell: pwsh
+
       - name: Checkout
         uses: actions/checkout@v4
 
-      - name: Build CLI
-        run: go build .
-        working-directory: ./cmd/river
+      - name: Prerequisites
+        run: |
+          if [ "$RUNNER_OS" == "Linux" ]; then
+            echo "RIVER_CMD_DIR=./cmd/river" >> $GITHUB_ENV
+          elif [ "$RUNNER_OS" == "Windows" ]; then
+            echo "RIVER_CMD_DIR=.\cmd\river" >> $GITHUB_ENV
+          fi
+        shell: bash
 
-      - name: Create database
-        run: psql --echo-errors --quiet -c '\timing off' -c "CREATE DATABASE river_dev;" ${ADMIN_DATABASE_URL}
+      - name: Build and install CLI
+        run: go install -v .
+        working-directory: ${{ env.RIVER_CMD_DIR }}
```

Comment:
`go install` instead of `go build` here means we don't have to set the working dir for every subsequent command.